k8s/airflow/airflow-app.yaml

```diff
@@ -19,6 +19,17 @@ spec:
         app: airflow
     spec:
       serviceAccountName: airflow-sa
+      initContainers:
+        - name: airflow-xcom
+          image: us.gcr.io/airqo-250220/airqo-apache-airflow-xcom
+          imagePullPolicy: Always
+          resources: {}
+          envFrom:
+            - configMapRef:
+                name: airflow-config
+          volumeMounts:
+            - name: xcom-config-volume
+              mountPath: /etc/airflow_xcom
```

Comment:
Added an init container to setup the airflow XCom before the schedular and webserver start. @123MwanjeMike